enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -160,50 +173,37 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 		}()
 	}()
 
-	r.copyContainerLogs(ctx, cid, result)
-	if result.Error != nil {
+	var stdout, stderr bytes.Buffer
+	eg := &errgroup.Group{}
+	eg.Go(func() error {
+		_, errCopy := stdcopy.StdCopy(&stdout, &stderr, hijackedResp.Reader)
+		return errCopy
```

Comment:
Wrap the error here instead of after `eg.Wait()`, since this func is the one that is copying container logs

+nit: can use the more conventional `err` here since there are no conflicting `err` variables in scope.

```suggestion
		_, err := stdcopy.StdCopy(&stdout, &stderr, hijackedResp.Reader)
		return wrapDockerError(err, "failed to copy container logs")
```